lib/cisco_node_utils/interface.rb

```diff
@@ -1094,10 +1103,14 @@ def switchport_vtp_feature?
       config_get('vtp', 'feature')
     end
 
-    def check_switchport_disabled
-      return if switchport_mode == :disabled || switchport_mode.nil?
```

Comment:
I was hoping we did not need the `switchport_mode.nil?` bit but looks like ti won't play nicely with XR.  New diffs coming...